.travis.yml

```diff
@@ -23,11 +23,13 @@ env:
     - PRODUCT_TESTS_SPECIFIC_ENVIRONMENT_2=true
     - HIVE_TESTS=true
     - KUDU_TESTS=true
+    - WEBUI_TESTS=true
```

Comment:
Let's rename this to `WEBUI_CHECKS` and move after `MAVEN_CHECKS`, since this is really a checker rather than an integration test.